recipe/meta.yaml

```diff
@@ -44,3 +45,4 @@ about:
 extra:
   recipe-maintainers:
     - ozagordi
+    - KingDarBoja
```

Comment:
@KingDarBoja Would you agree to help with the maintenance of this recipe in the future? It is not that much work and you did a great job updating the recipe just now!

Reply:
No problem mate, go ahead! 👍